Controllers/ProductController.js

```diff
@@ -0,0 +1,34 @@
+const DbService = require("../Services/DbService");
+const ProductService = require("../Services/ProductsService");
+const JsonResponseService = require("../Services/JsonResponseService");
+const DataCheckers = require("../Services/SanitiseValidateService");
+
+let getAllProducts = async(req, res) => {
```

Comment:
Should we be using const as the function should be static in what it returns i.e. a function